eth2/types/src/crosslink_committee.rs

```diff
@@ -1,7 +1,6 @@
 use crate::*;
-use tree_hash_derive::TreeHash;
 
-#[derive(Default, Clone, Debug, PartialEq, TreeHash)]
+#[derive(Default, Clone, Debug, PartialEq)]
```

Comment:
Had to delete this because of the panicky tree hash implementation for `Vec` that I deleted. We never used the implementation anyway.